dds_cli/dds_gui/app.py

```diff
@@ -11,7 +11,7 @@
 
 theme = Theme(
     name="custom",
-    primary="#3F3F3F",
+    primary="#666666",  # "#3F3F3F",
```

Comment:
Optional, could also be put in a new PR for just design.  

I'm a little unsure that the change I suggest should be here but: 
I think we should make the border and the text where it says "IMPORTANT INFORMATION" or "AUTHENTICATION" lighter. I like that the MOTD box ("card") is very visible now, but I suggest `A6A6A6` or something like that for this primary color.

**Currently I see this:**
<img width="681" height="1281" alt="image" src="https://github.com/user-attachments/assets/ac276dd6-d226-4670-9883-5fa0b6806c61" />


**Suggested change:**
<img width="916" height="1268" alt="image" src="https://github.com/user-attachments/assets/1ffed57b-ff78-496b-8a5b-6c7d4031cf1a" />